tests/dragonfly/replication_test.py

```diff
@@ -50,6 +50,7 @@ async def test_replication_all(df_local_factory, t_master, t_replicas, n_keys, n
 
     async def stream_data():
         """ Stream data during stable state replication phase and afterwards """
+        await asyncio.sleep(0)
         gen = gen_test_data(n_stream_keys, seed=2)
         for chunk in grouper(3, gen):
             await c_master.mset({k: v for k, v in chunk})
```

Comment:
We need better tests that emulate DB switches, but thats too much for this PR

I suggest keeping it as it, and fixing the potential bugs after we have better tests in the following PRs